File: src/main/java/model/msdk_w/Msdk_w.java
package model.msdk_w;

public class Msdk_w {
    public static void calculate() {

        double pspvdkug = 0.95;
        double wdkba_w = 0.5;
        double wdkugdn = 0.5;
        double kfmsnwdk = 1000;
        double msndko_w = 0; // ?

        double iat = 0;
        double ftvdk = 273/(iat+273);
        double klaf = 1.0;
        double fpvdkds_w = 1;

        double oneMinusWdkugdn = 1 - wdkugdn;
        double wdkbaMinusWdkugdn = Math.max(wdkba_w - wdkugdn, 0);
        double oneMinusPspvdkug = 1 - pspvdkug;

        double calc1 = ((oneMinusPspvdkug * wdkbaMinusWdkugdn)/oneMinusWdkugdn) + 1;
        double kfmsnwdkPlusMsndko_w = kfmsnwdk + msndko_w;
        double calc2 = calc1*kfmsnwdkPlusMsndko_w*ftvdk*klaf*fpvdkds_w;

        System.out.println(calc2);
    }
}

File: src/main/kotlin/domain/model/msdk_w/Msdk_w.kt
package domain.model.msdk_w

object Msdk_w {
    fun calculate() {
        val pspvdkug = 0.95
        val wdkba_w = 0.5
        val wdkugdn = 0.5
        val kfmsnwdk = 1000.0
        val msndko_w = 0.0

        val iat = 0.0
        val ftvdk = 273 / (iat + 273)
        val klaf = 1.0
        val fpvdkds_w = 1.0

        val oneMinusWdkugdn = 1 - wdkugdn
        val wdkbaMinusWdkugdn = maxOf(wdkba_w - wdkugdn, 0.0)
        val oneMinusPspvdkug = 1 - pspvdkug

        val calc1 = ((oneMinusPspvdkug * wdkbaMinusWdkugdn) / oneMinusWdkugdn) + 1
        val kfmsnwdkPlusMsndko_w = kfmsnwdk + msndko_w
        val calc2 = calc1 * kfmsnwdkPlusMsndko_w * ftvdk * klaf * fpvdkds_w
    }
}
